tests/test_module_cli.py

```diff
@@ -19,17 +23,20 @@ def test_parser_arg_errors(capsys):
 
 def test_rasters_list(monkeypatch):
     # Setup
-    monkeypatch.setattr(APIClient, '__init__', lambda s: None)
+    monkeypatch.setattr(APIClient, '__init__', _fake__init__)
     mock_rasterlist = MagicMock(return_value=['foo', 'bar'])
     mock_rasterlist.called is False
     monkeypatch.setattr(APIClient, 'list_rasters', mock_rasterlist)
     parse_args(['list', 'rasters'])
     mock_rasterlist.called is True
-
+    mock_rasterlist.reset_mock()
+    mock_rasterlist.called is False
```

Comment:
What does that line do? Actually, all of the `mock_rasterlist.called is ..`, do they work without `assert` ?

Reply:
Checked